TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficCaptureInputStream.java

```diff
@@ -0,0 +1,45 @@
+package org.opensearch.migrations.replay;
+
+import lombok.extern.slf4j.Slf4j;
+import org.opensearch.migrations.trafficcapture.protos.TrafficStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+@Slf4j
+public class TrafficCaptureInputStream implements ITrafficCaptureSource {
```

Comment:
...InputStream makes me expect to see a read() call that would return some data.  In this case, you're supplying data from read() calls.  You could call this TrafficCaptureSourceFromInputStream or something else.  As it is, it fits in the mold of FileInputStream, BufferedInputStream, etc.

Reply:
Renamed to hopefully be clear and not as wordy